server/handlers.go

```diff
@@ -375,11 +376,17 @@ func (s *Server) handleConnectorCallback(w http.ResponseWriter, r *http.Request)
 	}
 
 	if err != nil {
+		errMsg, errCode := "Failed to return user's identity.", http.StatusInternalServerError
+		// Prevent users not in an organization from logging into an application
+		if _, ok := err.(github.ErrForceOrgAddRequest); ok {
+			errMsg, errCode = "User is not an organization member.", http.StatusForbidden
```

Comment:
I wouldn't hard code this case. Other connectors, e.g. LDAP, will want to differentiate between internal errors and forbidden. If we're going to solve this we should do it in a general way. Probably out of scope for this PR.